libr/bin/format/mach0/mach0.c

```diff
@@ -3199,54 +3208,38 @@ static bool _check_if_debug_build(RBinFile *bf, struct MACH0_(obj_t) *mo) {
 	return false;
 }
 
-static void _r_bin_symbol_fini(void *_sym, void *user) {
-	RBinSymbol *sym = _sym;
-	if (sym) {
-		free (sym->name);
-		free (sym->libname);
-		free (sym->classname);
-	}
-}
-
-const RVector *MACH0_(load_symbols)(RBinFile *bf, struct MACH0_(obj_t) *mo) {
-	r_return_val_if_fail (bf, NULL);
-	if (!mo) {
-		return NULL;
-	}
+// R2_590 - should return bool
```

Comment:
remove, it returns a bool? but make the r_return_val_if_fail return false

Reply:
thx